### PR TITLE
Warlock config

### PR DESCRIPTION
**Pull request by dlemvigh**
Updated warlock config to BFA-prepatch.
Tested on level 110 warlock.

### CODE REVIEW

File: EventHorizon_Warlock/config.lua [hunk 1]
```diff
@@ -156,65 +242,36 @@ function EventHorizon:InitializeClass()
     cooldown = 196447,
     hasted = true,
     channeled = {196447, 15},
-    barcolors = {
-      barbgcolor  = { 0, 0.5, 0, 0.2 },
-      cooldown  = { 0, 1,   0, 0.5 },
-      casting    = { 0, 0,   0, 1   },
-      channeltick  = { 1, 0,   0, 1   }
-    }
-  })
```
Comment (Brusalk) [inline EventHorizon_Warlock/config.lua]: Is there a reason you removed the coloring from Channel Demonfire?

File: EventHorizon_Warlock/config.lua [hunk 2]
```diff
@@ -85,28 +125,70 @@ function EventHorizon:InitializeClass()
...
+  -- Soul Strike
+  self:newSpell({
+      requiredTree = 2,
+      requiredTalent = 11,
+      cooldown = 264057
+  })
+
+  -- Nether Portal
+  self:newSpell({
+      requiredTree = 2,
```
Comment (Brusalk) [inline EventHorizon_Warlock/config.lua]: Can you please make the indentation here 2 spaces rather than 4? 😄

Comment (dlemvigh) [reply EventHorizon_Warlock/config.lua]: I'll fix the indentation

File: EventHorizon_Warlock/config.lua (hunk 1 above)
Comment (dlemvigh) [reply EventHorizon_Warlock/config.lua]: Aesthetics mostly.
The previous color scheme used pure red/green/blue.
I preferred the default color scheme.

If you want I can show a before and after screenshot.

**Pull request merged**